src/runtime/node/module/$cloudflare.ts

```diff
@@ -16,7 +16,10 @@ export {
   _resolveFilename,
   _resolveLookupPaths,
   builtinModules,
+  constants,
```

Comment:
@anonrig I think you have plan to enable compile cache, those methods are not implemented in workerd yet, right?
`enableCompileCache`, `getCompileCacheDir`, and related `constants`

Reply:
It is enabled right now. And yes these methods are not implemented.